src/test/ui/consts/min_const_fn/allow_const_fn_ptr.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(rustc_attrs, rustc_const_unstable, const_fn, staged_api)]
```

Comment:
This (and the other file, `allow_const_fn_ptr_feature_gate.rs`) should not need `rustc_const_unstable` since it ain't used.

Also please extend this test with a file that doesn't use `const_fn`